package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "radiance-ui",
-  "version": "13.0.0-alpha.4",
+  "version": "13.0.0-beta.0",
```

Comment:
I probably should've created a separate branch outside of `master` for the beta release but it's not a big deal to have this in flux.